greenplumpython/experimental/embedding.py

```diff
@@ -0,0 +1,204 @@
+from collections import abc
+from typing import Any, Callable, cast
+from uuid import uuid4
+
+import greenplumpython as gp
+from greenplumpython.func import FunctionExpr
+from greenplumpython.row import Row
+
+
+@gp.create_function
+def _generate_embedding(content: str, model_name: str) -> gp.type_("vector", modifier=384):  # type: ignore reportUnknownParameterType
+    import sys
```

Comment:
Yes, we should definitely limit the degree of parallelism, otherwise the task won't end when the number of data increases. The number will depends on number of cores and memory, if we want a constant number  4 seems fine to me.